FontAwesome/FontAwesome.swift

```diff
@@ -33,19 +33,29 @@ public struct FontAwesomeConfig {
 
     /// Taken from FontAwesome.io's Fixed Width Icon CSS.
     public static let fontAspectRatio: CGFloat = 1.28571429
+
+    /// Whether Font Awesome Pro fonts should be used (not included).
+    ///
+    /// To use Font Awesome Pro fonts, you should add these to your main project and
+    /// make sure they are added to the target and are included in the Info.plist file.
+    public static var useProFonts: Bool = false
```

Comment:
Nit: Could you change this to `usesProFonts`?

Reply:
Done